authconfig.ks.in

```diff
@@ -0,0 +1,40 @@
+#version=DEVEL
+#test name: authconfig
+%ksappend repos/default.ks
+install
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+clearpart --all --initlabel
```

Comment:
Shouldn't you use fragments for all these default settings?

Reply:
Good point, https://github.com/rhinstaller/kickstart-tests/pull/312/commits/90f6953d43d26cb41d0df5f4c78cddb87ef949f1